Documentation/Manual/LaTeX/chap-first-application.tex

```diff
@@ -109,24 +160,27 @@ \section{A First Attempt}
 
 \verbatiminput{ex1.lisp}
 
-As we can see in this example, we have put our application in a separate
-package, here a package named \texttt{APP}.  While not required, putting the
-application in its own package is good practice.
+As we can see in this example, we have put our application in a
+separate package, here a package named \texttt{my-first-app}. While
+not required, putting the application in its own package is good
+practice.
 
 The package for the application uses two packages: \package{CLIM} and
 \package{CLIM-LISP}.  The \package{CLIM} package is the one that
-contains all the symbols needed for using CLIM.  The \package{CLIM-LISP}
-package replaces the \package{COMMON-LISP} package for CLIM
-applications.  It is essentially the same as the \package{COMMON-LISP}
-package as far as the user is concerned.
+contains all the symbols needed for using CLIM.  The
+\package{CLIM-LISP} package replaces the \package{COMMON-LISP} package
+for CLIM applications.  It is essentially the same as the
+\package{COMMON-LISP} package as far as the user is concerned.
 
 In our example, we export the symbol that corresponds to the main
-function to start our application, here called \texttt{APP-MAIN}.
+function to start our application, here called
+\texttt{run-my-first-app}.
 
```

Comment:
From personal experience when learning a new language or library, it is sometimes difficult to differentiate between what names are part of the library or something that a user should provide. Imagine something called 'display-application-main-frame'. If you know nothing about CLIM, is it part of the spec, or a user-provider name?
That's why I use that sort of very clear names to show what is what.